src/emscripten-optimizer/optimizer-shared.cpp

```diff
@@ -67,7 +56,7 @@ AsmType detectType(Ref node, AsmData *asmData, bool inVarDef, IString minifiedFr
   switch (node[0]->getCString()[0]) {
     case 'n': {
       if (node[0] == NUM) {
-        if (!isInteger(node[1]->getNumber())) return ASM_DOUBLE;
+        if (!wasm::isInteger(node[1]->getNumber())) return ASM_DOUBLE;
```

Comment:
Yes, there was just one (aside from me) to that directory. But the problem isn't symmetrical - MIT is more permissive than Apache, there isn't a problem to include the emscripten code here, just like you can include BSD code in GPL projects, but not vice versa. (Except one can't apply new copyright headers, of course.)

Sure, sounds fine to fix it for now - but in what way? If it's fixing it temporarily in a way that doesn't make code in `emscripten_optimizer/` look outside, then then sounds ok.